### ISSUE TITLE
Export JsonDecodeError

### ISSUE THREAD

**Issue by mlegenhausen** [OliverJAsh/decode-ts#4]
It would be nice if `JsonDecodeError` would be added to the public interface. That would allow me to reuse existing errors or even better to extend the union type for longer parsing pipelines. Currently I need to export it from `'decode-ts/target/types'`

**Comment by OliverJAsh** [comment]
Hi @mlegenhausen! So you would like to be able to do

``` ts
import { JsonDecodeError } from 'decode-ts'
```

instead of


``` ts
import { JsonDecodeError } from 'decode-ts/target/types'
```

?

I guess we can just add `export * from './types` to `index.ts`.

**Comment by mlegenhausen** [comment]
@OliverJAsh Yes that would be great.

**Comment by OliverJAsh** [comment]
Fixed in v0.0.11 https://github.com/OliverJAsh/decode-ts/commit/0a2eca22635359794bbe6137a61f4b3a7e428a5f

**Comment by mlegenhausen** [comment]
@OliverJAsh Thanks alot
